src/Microsoft.AspNetCore.Mvc.Core/Internal/ControllerActionInvoker.cs

```diff
@@ -24,6 +24,7 @@ public class ControllerActionInvoker : ResourceInvoker, IActionInvoker
 
         private ActionExecutingContext _actionExecutingContext;
         private ActionExecutedContext _actionExecutedContext;
+        private const string ActionFilter = "Action Filter";
```

Comment:
There seem to be 3 styles for these constants in this PR:
1. This case here, with a member `cost`
2. The `FilterCategoryConstants` nested type
3. Using string literals, e.g. `Page Filter`

I think using `const`s is good, so maybe just change `Page Filter` to a `const` and then at least it feels more consistent?